chapter-11/recipe-04/cxx-example/conda-recipe/meta.yaml

```diff
@@ -7,13 +7,16 @@ source:
 
 build:
   number: 0
+  binary_relocation: true
   script:
-    - cmake -H. -Bbuild_conda -DCMAKE_INSTALL_PREFIX=${PREFIX}
+    - cmake -H. -Bbuild_conda -G "${CMAKE_GENERATOR}" -DCMAKE_INSTALL_PREFIX=${PREFIX}         # [not win]
+    - cmake -H. -Bbuild_conda -G "%CMAKE_GENERATOR%" -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" # [win]
```

Comment:
How does it know which one to take?

Reply:
They're called selectors and don't play well with multiline commands.

Reply:
Cool!